### PR TITLE
Update consumer RL tutorial

### PR DESCRIPTION
**Pull request by lena-larionova**
* Break out tutorial steps into headings
* Adding more context for each step
* Clean up

https://deploy-preview-72--kongdeveloper.netlify.app/how-to/add-rate-limiting-for-a-consumer-with-kong-gateway/

### CODE REVIEW

**Comment by lena-larionova** [comment]
I struggled do decide how much context to provide for each step. How many words are too many?

Comparing to https://github.com/Kong/developer.konghq.com/pull/70, this is a lot more minimal. At the same time, the tiers tutorial is definitely more complex and requires more explanation.

**Comment by fabianrbz** [comment]
I think it's enough, this one has less complexity, and the steps provide enough context to understand why they are needed

File: app/_how-tos/add-rate-limiting-for-a-consumer-with-kong-gateway.md [hunk 1]
```diff
@@ -61,25 +65,28 @@ entities:
...
-{% endcapture %}
-{{ plugin | indent: 3 }}
 
-1. Create a [consumer](/gateway/entities/consumer/) with a key.
+## 2. Create a consumer
+
+Consumers let you identify the client that's interacting with Kong Gateway. 
+With the Key Authentication plugin enabled globally, the consumer needs an API key to access any Kong Gateway services.
+
+Append the following snippet to the `kong.yaml` file to create a [consumer](/gateway/entities/consumer/) with an API key:
```
Comment (fabianrbz) [inline app/_how-tos/add-rate-limiting-for-a-consumer-with-kong-gateway.md:74]: the `entity_examples` block renders a copy if `append_to_existing_section: true` , and this looks a bit weird
Screenshot 2024-08-29 at 18 10 27

Comment (lena-larionova) [reply app/_how-tos/add-rate-limiting-for-a-consumer-with-kong-gateway.md:74]: I noticed that too - let me remove my line.

**Pull request merged**